api/api_sources/sources/libs/core-database/baseSchema.ts

```diff
@@ -431,6 +492,22 @@ export class  BaseSchema {
     public static get id(): string {
         return this.shared.table.columns.id;
     }
+
+    /**
+     * @description Get all migration sql file related to schema
+     * @returns sting []
```

Comment:
string (if we want to be really picky...)

Reply:
Fixed